src/test/test_namespaces_pattern.py

```diff
@@ -40,25 +40,25 @@ def setUp(self):
             "cisa",
             "custom",  # not in enum, but valid for the pattern
             "abc",  # not in enum, but valid for the pattern
-            "x_com.example#foo/",
-            "x_com.example#foo//.org.example#bar",
+            "x_example.test#test/",
+            "x_example.test#test//.org.example#bar",
             "ssvc/de-DE/.org.example#reference-arch-1",  # valid BCP-47 tag, reverse domain notation, hash
             "ssvc//.de.bund.bsi$de-DE",  # BSI's translation of SSVC
             "ssvc//.de.bund.bsi#ref-arch-1/de-DE",  # BSI's official translation to German as used in Germany of its ref-arch-1 model which is originally written in English
             "ssvc//.de.bund.bsi#ref-arch-2$de-DE",  # BSI's ref-arch-2 model which is originally written in German
             "ssvc//.de.bund.bsi#ref-arch-2$de-DE/en-GB",  # BSI's official translation to English as used in GB of its ref-arch-2 model which is originally written in German
-            "ssvc//.example.organization#model/.com.example#foo",  # empty BCP-47 tag, two segments with one hash each
+            "ssvc//.example.organization#model/.example.test#test",  # empty BCP-47 tag, two segments with one hash each
             "nist#800-30",  # NIST's registered model regarding its publication 800-30
             "x_gov.nist#800-30/de-DE",  # NIST's official translation to German as used in Germany of its model (regarding its publication) 800-30
             "x_gov.nist#800-30//.de.bund.bsi$de-DE",  # BSI's translation to German as used in Germany of NIST's model (regarding its publication) 800-30
-            "x_com.example.test#bar/pl-PL/.com.example#foo/.org.example#bar/newfound",  # valid BCP-47 tag and multiple segments
-            "com.example",  # valid namespace with dots following reverse domain notation
-            "x_com.example#foo",  # valid namespace with x_ prefix and dots following reverse domain notation
+            "x_example.test.test#test/pl-PL/.example.test#test/.example.test#test/newfound",  # valid BCP-47 tag and multiple segments
```

Comment:
This example was too much replacement...
```suggestion
            "x_example.test.test#test/pl-PL/.example.test#another-collection/.org.example#a-different-collection/en-CA-newfound",  # valid BCP-47 tag and multiple segments